src/transformers/models/pegasus/tokenization_pegasus.py

```diff
@@ -226,10 +226,10 @@ def _convert_id_to_token(self, index: int) -> str:
             token = self.sp_model.IdToPiece(index - self.offset)
         return token
 
-    def convert_tokens_to_string(self, tokens):
-        """Converts a sequence of tokens (string) in a single string."""
-        out_string = self.sp_model.decode_pieces(tokens)
-        return out_string
+    # def convert_tokens_to_string(self, tokens):
+    #     """Converts a sequence of tokens (string) in a single string."""
+    #     out_string = self.sp_model.decode_pieces(tokens)
+    #     return out_string
```

Comment:
Nit, I think it fell through the cracks :smile:

Reply:
Definitely did! 😅